src/PythonMigrationViewExtension/PythonMigrationViewExtension.csproj

```diff
@@ -37,6 +37,9 @@
     <Reference Include="DiffPlex.Wpf, Version=1.1.1.0, Culture=neutral, PublicKeyToken=1d35e91d1bd7bc0f, processorArchitecture=MSIL">
       <HintPath>..\packages\DiffPlex.Wpf.1.1.1\lib\net46\DiffPlex.Wpf.dll</HintPath>
     </Reference>
+    <Reference Include="Microsoft.Practices.Prism, Version=4.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
```

Comment:
where is this coming from?

Reply:
no idea, wasn't showing up in NuGet packages list in VS either, removed it and builds just fine.